app/src/main/java/com/codesoom/assignment/AssignmentHttpHandler.java

```diff
@@ -0,0 +1,84 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.models.Task;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.*;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class AssignmentHttpHandler implements HttpHandler {
+    private ObjectMapper objectMapper = new ObjectMapper();
+
+    private List<Task> tasks = new ArrayList<>();
+
+//    public AssignmentHttpHandler() {
+//        Task task = new Task();
+//        task.setId(1L);
+//        task.setTitle("Do nothing...");
+//
+//        tasks.add(task);
+//    }
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        String method = exchange.getRequestMethod();
+        URI uri = exchange.getRequestURI();
+        String path = uri.getPath();
+
+        InputStream inputStream = exchange.getRequestBody();
+        String body = new BufferedReader(new InputStreamReader(inputStream))
+                        .lines()
+                        .collect(Collectors.joining("\n"));
+
+        System.out.println(method + " " + path);
+        // body 없는 경우 넘어가기
+        if (!body.isBlank()) {
+            // test object 확인
+            Task task = toTask(body);
+            System.out.println(task);
+            System.out.println(body);
+        }
+
+        String content = "Hello, world!";
+
+        if (method.equals("GET") && path.equals("/tasks")) {
```

Comment:
여기에선 문제가 없을 수 있지만! 
문자열에서 `equals` 를 사용할 때는 상수값이 앞에 있는 것이 좋아요! 
왜냐하면 상수는 `null` 이 아니기 때문이죠!

Reply:
상수값과 비교하라는 말씀이신가요? 문자열 비교는 equals 가 적합하지 않나요?

Reply:
```suggestion
"GET".equals(method)
```
이런 형식이 되는 것이 더 안전하다는 뜻이었습니다!

Reply:
```java
"GET".equals(method)
```
이런 형식이 되는 것이 더 안전하다는 뜻이었습니다!

Reply:
와 실제 예제 정말 감사합니다. 이해가 되었어요!!